vega-lite-schema.json

```diff
@@ -37,6 +37,72 @@
             ],
             "type": "string"
         },
+        "AreaConfig": {
```

Comment:
I disagree.  Schema diff is a great way to check if your PR / commit reports all the diff correctly (and we can copy the changes to release log easily.) 

Instead, I argue that we should always commit them in a PR.  Merge conflicts is easy to fix enough that it doesn't hurt to be consistent everytime.